docs/root/configuration/observability/access_log.rst

```diff
@@ -410,12 +410,15 @@ The following command operators are supported:
     JSON struct or list is rendered. Structs and lists may be nested. In any event, the maximum
     length is ignored
 
-%FILTER_STATE(KEY):Z%
+%FILTER_STATE(KEY:F*):Z%
   HTTP
     :ref:`Filter State <arch_overview_data_sharing_between_filters>` info, where the KEY is required to
     look up the filter state object. The serialized proto will be logged as JSON string if possible.
     If the serialized proto is unknown to Envoy it will be logged as protobuf debug string.
     Z is an optional parameter denoting string truncation up to Z characters long.
+    F is an optional parameter used to indicate which method FilterState uses for serialization. 
+    If 'PLAIN' is set, FilterState will call 'serializeAsString' to output an unstructured string. 
```

Comment:
```suggestion
    If 'PLAIN' is set, the filter state object will be serialized as an unstructured string. 
```
Same below, i.e. best not to talk about internal implementation details that might change in user facing docs.